src/wcmValidateDevice.c

```diff
@@ -332,7 +337,6 @@ int wcmDeviceTypeKeys(InputInfoPtr pInfo)
 		case 0x351:/* Cintiq Pro 24 */
 		case 0x352:/* Cintiq Pro 32 */
 		case 0x37C:/* Cintiq Pro 24 Pen-Only */
-		case 0x390:/* Cintiq 16 */
 			TabletSetFeature(priv->common, WCM_ROTATION);
```

Comment:
I believe this is the right spot for all of these tablets. The fall through here sets only `WCM_LCD` and `WCM_ROTATION`.

Reply:
Oh, focused on INPUT_PROP_DIRECT too much and stopped at the first WCM_LCD. Will push the fix.